code/modules/reagents/reagent_containers.dm

```diff
@@ -33,6 +33,9 @@
 	RegisterSignal(reagents, COMSIG_PARENT_QDELETING, .proc/on_reagents_del)
 
 /obj/item/reagent_containers/Destroy()
+	for(var/atom/movable/item as anything in contents)
+		item.forceMove(drop_location())
+	contents = null
```

Comment:
this is not good practice on `Destroy()`
use the object deconstruction procs instead and take into account whether they are being properly deconstructed or not